README.md

```diff
@@ -92,12 +95,16 @@ except for `libxmlsec1`, which you can install with Homebrew:
 
 ### Running Tests
 
+Make sure you have [PhantomJS](http://phantomjs.org) installed prior to running tests. Install with the following:
+
+    $ brew install phantomjs
```

Comment:
failure is pretty clear when you don't have phantomjs installed; @sbc100 any suggestions on how to specify OSX specific steps?